src/query/ObjectSelector.js

```diff
@@ -78,7 +91,7 @@ export default class ObjectSelector {
     if (aoiDataSource.entities.contains(entity)) {
       const aoiElement = document.querySelector('ngm-aoi-drawer');
       attributes = aoiElement.getInfoProps({...props, name: entity.name});
-    } else if (earthquakesDataSource.entities.contains(entity)) {
+    } else if (earthquakesDataSource && earthquakesDataSource.entities.contains(entity)) {
```

Comment:
I think you can use the ".?" operator for that.

Reply:
We should add an additional loader for this syntaxis